components/webdriver_server/lib.rs

```diff
@@ -189,51 +189,51 @@ struct Handler {
 }
 
 #[derive(Clone, Copy, Debug, PartialEq)]
-enum ServoExtensionRoute {
-    GetPrefs,
-    SetPrefs,
-    ResetPrefs,
+enum ServoExtensionRoutePrefs {
+    Get,
+    Set,
+    Reset,
 }
```

Comment:
Hrm...I think these should probably stay. I'm not sure that `ServoExtensionRoutePrefs` makes sense...it's just that all of the ExtensionRoutes are currently pref related at the moment.

Reply:
You're right, it was cleaner originally. I reverted it back and added an allow.